uvm/js/common/ungrid/GridController.js

```diff
@@ -467,10 +467,11 @@ Ext.define('Ung.cmp.GridController', {
                                     }
                                }
                             }
-        
+
+                            
                             // Currently, custom validator is tailored exclusively for the WG App.
                             // To extend its functionality to other apps, ensure custom validators across each app retrieve stored values during import operations.
-                            if(record.javaClass === 'com.untangle.app.wireguard_vpn.WireGuardVpnTunnel' || record.javaClass === 'com.untangle.app.openvpn.OpenVpnRemoteClient' || record.javaClass === 'com.untangle.app.openvpn.OpenVpnGroup' || record.javaClass === "com.untangle.app.openvpn.OpenVpnExport" || record.javaClass === "com.untangle.app.openvpn.OpenVpnConfigItem" || record.javaClass === "com.untangle.app.ipsec_vpn.IpsecVpnTunnel" || record.javaClass === "com.untangle.app.ipsec_vpn.IpsecVpnNetwork"  ){
+                            if(record.javaClass === 'com.untangle.app.wireguard_vpn.WireGuardVpnTunnel' || record.javaClass === 'com.untangle.app.openvpn.OpenVpnRemoteClient' || record.javaClass === 'com.untangle.app.openvpn.OpenVpnGroup' || record.javaClass === "com.untangle.app.openvpn.OpenVpnExport" || record.javaClass === "com.untangle.app.openvpn.OpenVpnConfigItem" || record.javaClass === "com.untangle.app.ipsec_vpn.IpsecVpnTunnel" || record.javaClass === "com.untangle.app.ipsec_vpn.IpsecVpnNetwork" || record.javaClass ==="com.untangle.app.tunnel_vpn.TunnelVpnRule" ){
```

Comment:
Instead of a long list of Java classes to compare, let's create an array to compare the record.javaclass.

Reply:
Done